nautobot/ipam/tests/test_filters.py

```diff
@@ -603,25 +601,25 @@ def test_filter_address(self):
 
     def test_mask_length(self):
         # Test filtering by a single integer value
-        params = {"mask_length": self.queryset.first().mask_length}
+        params = {"mask_length": [self.queryset.first().mask_length]}
         self.assertQuerysetEqualAndNotEmpty(
-            self.filterset(params, self.queryset).qs, self.queryset.filter(mask_length=params["mask_length"])
+            self.filterset(params, self.queryset).qs, self.queryset.filter(mask_length__in=params["mask_length"])
         )
```

Comment:
Is this set of changes necessary? Does it represent a loss of functionality (before could be a single value or a list, now must always be a list)?

Reply:
Enhanced `multivalue_field_factory()` to handle passing in a single integer as well since this field is mapped to `MutliValueNumberFilter`